examples/train_dagger_atari_interactive_policy.py

```diff
@@ -0,0 +1,41 @@
+"""Training DAgger with an interactive policy that queries the user for actions.
```

Comment:
If a matplotlib GUI backend isn't installed, it'll fail with a somewhat cryptic error:

```/home/adam/dev/imitation/src/imitation/policies/interactive.py:99: UserWarning: Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.
  fig.show()
```

and indeed no figure displays.

Installing the relevant backend seems out-of-scope for this project. But might want to check if the backend is interactive (I think [plt.isinteractive()](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.isinteractive.html#matplotlib.pyplot.isinteractive) checks for this) and warn if not with link to relevant docs e.g. https://matplotlib.org/stable/users/explain/backends.html

Reply:
Could you please check again what message you are getting and if this is an error or a warning? In my case, when I set a non-GUI backend like `Agg`, I get a warning like this (and the execution continues):
```
UserWarning: Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.
  fig.show()
```
Actually, `plt.isinteractive()` checks for the interactive mode; GUI backends like MacOsX can be in both modes (and we do not need to turn on interactive for it to work). What we actually would like is to check if the backend is "GUI" or "non-GUI" but from a simple search, it does not seem like there is a nice way to do it (rather than check with some white-list of backends). Given that, and the fact that the message I listed above is not that bad, I'd keep this as-is for now. Alternatively, we could opt for throwing an error/assert instead of warning, but again this would require a white-list of backends. WDYT?

Reply:
For additional context: on my laptop, the example runs nicely although the mode is not interactive by default.